fe/fe-core/src/main/java/com/starrocks/analysis/FunctionCallExpr.java

```diff
@@ -85,6 +87,16 @@
                     .add("stddev").add("stddev_val").add("stddev_samp")
                     .add("variance").add("variance_pop").add("variance_pop").add("var_samp").add("var_pop").build();
 
+    private static final ImmutableSet<String> UTF8_STRING_FUNCTION_SET =
+            new ImmutableSortedSet.Builder(String.CASE_INSENSITIVE_ORDER)
```

Comment:
All function name should be lower case， and needn't CASE_INSENSITIVE.

Reply:
no CASE_INSENSITIVE constant